src/ecdsa/ecdh.py

```diff
@@ -54,9 +54,9 @@ def __init__(self, curve=None, private_key=None, public_key=None):
         self.private_key = None
         self.public_key = None
         if private_key:
-            self.load_private_key(private_key)
+            self.local_pk_set(private_key)
         if public_key:
-            self.load_received_public_key(public_key)
+            self.remote_vk_set(public_key)
```

Comment:
why did you change them?

Reply:
method names more consistent with `keys.py`

Reply:
I was talking about the `_str` vs `_bytes`; changing it to `_bytes` was what was ok with, so I didn't reiterate it in https://github.com/warner/python-ecdsa/pull/149#pullrequestreview-314508637

Reply:
ok. ill revert changes and start to do docstrings. ok?

Reply:
ok